UnityProject/Assets/Scripts/Objects/Pipes/Devices/AirVent.cs

```diff
@@ -43,14 +45,6 @@ public override void TickUpdate()
 
 		private void CheckAtmos()
 		{
-			// FIXME I'm just handling the exception here, I'm no atmos nerd so I don't know what's happening.
-			// maybe it is just an initialization order problem?
-			if (metaNode == null)
-			{
-				Logger.LogError("Airvent found metadaNode to be null. Returning with no op.", Category.Atmos);
```

Comment:
this may spam?

Reply:
I did that, it didn't spam for me

Reply:
It was an initialization problem so it only happened during round start, it would also break atmos on editor for me without the null check

Reply:
also Aran is removing the log, not adding it